plugin/pki/path_import_queue.go

```diff
@@ -150,9 +152,20 @@ func (b *backend) fillImportQueueTask(roleName string, noOfWorkers int, storage
 }
 
 func (b *backend) importToTPP(storage logical.Storage, conf *logical.BackendConfig) {
+
+	timeout := 1
+	env := os.Getenv("VAULT_VENAFI_IMPORT_TIMEOUT")
```

Comment:
Firstly, I think `timeout` is not god name. Secondly - is configuring something via environment variable a best practice?